Formula/stringtie.rb

```diff
@@ -1,7 +1,7 @@
 class Stringtie < Formula
   # cite Pertea_2015: "https://doi.org/10.1038/nbt.3122"
   desc "Transcript assembly and quantification for RNA-Seq"
-  homepage "https://github.com/gpertea/stringtie"
+  homepage "https://ccb.jhu.edu/software/stringtie"
```

Comment:
```suggestion
  homepage "https://ccb.jhu.edu/software/stringtie/"
```

Reply:
https://ccb.jhu.edu/software/stringtie redirects to the insecure URL http://ccb.jhu.edu/software/stringtie/
```console
$ curl -is https://ccb.jhu.edu/software/stringtie | grep Location
Location: http://ccb.jhu.edu/software/stringtie/
```
It works with the trailing slash: https://ccb.jhu.edu/software/stringtie/